src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/Example3.java

```diff
@@ -0,0 +1,43 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="JavadocParagraph"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocparagraph;
+
+// xdoc section -- start
+// violation 6 lines below '<p> tag should not precede HTML block-tag '<h1>''
+// violation 9 lines below '<p> tag should not precede HTML block-tag '<pre>''
+// violation 11 lines below '<p> tag should not precede HTML block-tag '<ul>''
+/**
+ * Some summary.
+ *
+ * <p><h1>P tag is preceding block-tag h1, not allowed</h1>
```

Comment:
we need to make examples to be same by code and different only by violation comments.
please align all examples by target content and line numbers see rules at https://github.com/checkstyle/checkstyle/issues/13345